### PR TITLE
[metricbeat]Add kubernetes scheduler dashboard

### PR DESCRIPTION
**Pull request by odacremolbap**
Add kubernetes scheduler dashboard

### CODE REVIEW

**Comment by odacremolbap** [comment]
image

File: metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-scheduler.json [hunk 1]
```diff
@@ -0,0 +1,847 @@
...
+                }
+              }
+            ],
+            "query": {
+              "language": "kuery",
+              "query": ""
+            }
+          }
+        },
+        "title": "Kubernetes - Host selector controller manager",
```
Comment (jsoriano) [inline metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-scheduler.json]: We follow this schema for visualization titles:
```suggestion
        "title": "Host selector controller manager [Metricbeat Kubernetes]",
```

Once a visualization is placed in a dashboard, its title can be customized, and then there is no need to mention the module as the dasboard is already titled with that. So in the dashboard this visualization could be titled `Host selector controller manager`.

Comment (jsoriano) [inline metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-scheduler.json]: Should this mention "controller manager"?

File: metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-scheduler.json [hunk 2]
```diff
@@ -0,0 +1,847 @@
...
+                  "params": {
+                    "query": "kubernetes.controllermanager"
+                  },
+                  "type": "phrase",
+                  "value": "kubernetes.controllermanager"
+                },
+                "query": {
+                  "match": {
+                    "event.dataset": {
+                      "query": "kubernetes.controllermanager",
```
Comment (jsoriano) [inline metricbeat/module/kubernetes/_meta/kibana/7/dashboard/Metricbeat-kubernetes-scheduler.json]: Is controller manager expected to appear here?

**Comment by odacremolbap** [comment]
@jsoriano this is not the json you are reviewing is not the one I should be pushing.
this is in progress, and I'm testing removing all contents from the dasboard (CI succeed) or adding some very brief and almost random content (CI fails)

still trying to sort out the beat magic commands to get this right

**Comment by odacremolbap** [comment]
I'm very sorry to request the review again, but I've added:

- 30s interval to markdown visualization, since it was going blank often
- CPU + Memory visualization (to make this par with `proxy` dashboard)
- Docs primer (will create an issue to improve once `controller-manager` and `proxy` are merged)

**Comment by odacremolbap** [comment]
jenkins, test this my dear

**Comment by odacremolbap** [comment]
jenkins, test this

**Comment by sayden** [comment]
Tested locally
image

**Pull request merged**